tests/test_main.py

```diff
@@ -156,10 +171,12 @@ def test_distance_class_iadd_number(kilometers, kilometers2, result):
     [
         (50, 3, 150),
         (30, 7, 210),
-        (45, 5, 225)
+        (45, 5, 225),
+        (1.5, 3.0, 4.5),
```

Comment:
The expected result for the multiplication of `Distance(1.5)` by `3.0` should be `4.5`, not `4.5`. Please correct the expected result to ensure the test case is accurate.